javaparser-core/src/main/java/com/github/javaparser/ast/visitor/DumpVisitor.java

```diff
@@ -1587,8 +1587,7 @@ public void visit(LambdaExpr n, Object arg) {
         printJavaComment(n.getComment(), arg);
 
         List<Parameter> parameters = n.getParameters();
-		boolean printPar = false;
-		printPar = n.isParametersEnclosed();
+		boolean printPar = n.isParametersEnclosed();
```

Comment:
If I start making everything final that can be final, I will submit another huge patch :)